### DISCUSSION TITLE
R installation problem in Vscode.

### DISCUSSION THREAD

**Discussion by donkrue** [JanDeDobbeleer/oh-my-posh#4489] [Troubleshoot]
Friends, please help me to solve the problem I am having while installing R extension.
![Screenshot (23)](https://github.com/JanDeDobbeleer/oh-my-posh/assets/119876143/15f2a4b3-85cf-4f3f-a71c-95e5c654cfbf)

**Comment by JanDeDobbeleer** [comment]
@donkrue what does this have to do with oh-my-posh?

**Reply by AjayKMehta** [reply]
Even the problem statement is incorrect.You are trying to call R code in PowerShell. You need to have a R terminal to do so.

@donkrue Please see https://code.visualstudio.com/docs/languages/r
